client/src/partials/depots_management/depots_management.html

```diff
@@ -0,0 +1,121 @@
+                <th>{{ "DEPOT.IS_WAREHOUSE" | translate }}</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="depot in DepotCtrl.depotList">
+                <td>{{ depot.reference }}</td>
+                <td>{{ depot.text }}</td>
+                <td  class="text-center">
+                  <i ng-if="depot.is_warehouse" class="glyphicon glyphicon-ok-sign"></i>
+                </td>
+                <td class="text-center">
+                  <a name="depot-{{depot.text}}" class="btn btn-xs btn-default"
+                    ng-click="DepotCtrl.update(depot.uuid)">
+                    <i class="glyphicon glyphicon-pencil"></i> {{ 'FORM.EDIT' | translate }}
+                  </a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+
+      <div class="col-md-6" ng-switch="DepotCtrl.view">
+
+        <div class="alert alert-info" ng-switch-default>
+          <h4>{{ "DEPOT.DEPOTS_MANAGEMENT" | translate }}</h4>
+          <p>{{ "DEPOT.DESCRIPTION" | translate }}</p>
+        </div>
+
+        <div ng-switch-when="action">
+          <form class="panel panel-primary" name="ActionForm" ng-submit="DepotCtrl.submit(ActionForm.$invalid)" novalidate>
+            <div class="panel-heading">
+              {{ "DEPOT.ADD" | translate }}
+            </div>
+            <div class="panel-body">
+
+              <div class="form-group" ng-class="{ 'has-error' : ActionForm.$submitted && ActionForm.text.$invalid }">
+                <label class="control-label">
+                  {{ "COLUMNS.NAME" | translate }}
+                </label>
+                <input class="form-control" name="text" ng-model="DepotCtrl.depot.text" required>
+                <div class="help-block" ng-messages="ActionForm.text.$error" ng-show="ActionForm.$submitted">
+                  <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+                </div>
+              </div>
+
+              <div class="form-group">
+                <label class="control-label">
+                  <input
+                    type="checkbox"
+                    name="is_warehouse"
+                    ng-true-value="1"
+                    ng-false-value="0"
+                    ng-model="DepotCtrl.depot.is_warehouse">
+                  {{ "DEPOT.IS_WAREHOUSE" | translate }}
+                </label>
+              </div>
+
+            </div>
+            <div class="panel-footer">
+
+              <div class="form-group">
+                <button class="btn btn-default pull-right" type="submit" data-method="submit">
+                  {{ "FORM.SUBMIT" | translate }}
+                </button>
+                <button class="btn btn-default" type="button" ng-click="DepotCtrl.cancel()" data-method="cancel">
+                  {{ "FORM.CANCEL" | translate }}
+                </button>
+                <div class="clearfix"></div>
```

Comment:
I used this class because without it the `error feedback` is too near buttons, the visual aspect is not very good

![2016-03-05_174450](https://cloud.githubusercontent.com/assets/5445251/13548858/045dd1bc-e2fa-11e5-8de0-aa975c8ca130.png)